src/components/FeeLabelToggle/FeeLabelToggle.stories.js

```diff
@@ -20,16 +20,33 @@
 
 import * as React from 'react';
 import { storiesOf } from '@storybook/react-native';
+import { defaultFiatCurrency } from 'constants/assetsConstants';
 
-import FeeLabelToggle from './FeeLabelToggle';
+import { FeeLabelToggleComponent as FeeLabelToggleStoryItem } from './FeeLabelToggle';
```

Comment:
It seems that there is no reason for renaming import.

Reply:
@mdjastrzebski Thank you for reviewing.The changes has been done.